src/mrsimulator/__init__.py

```diff
@@ -144,9 +388,49 @@ def parse(py_dict, parse_units: bool = True):
         else [SignalProcessor() for _ in sim.methods]
     )
 
-    params = None
-    if "params" in py_dict:
-        val = py_dict["params"]
-        params = None if val is None else Parameters().loads(s=val)
+    application = py_dict["application"] if "application" in py_dict else {}
+
+    return sim, signal_processors, application
+
 
-    return sim, signal_processors, params
+def to_new_mrsim(filename: str, overwrite: bool = False):
```

Comment:
Was thinking for this to be a public function in case anyone tries to load a v0.6 file. Easier to point users to this instead of showing them how to reconstruct their file.